erpnext/agriculture/doctype/strain/strain.py

```diff
@@ -32,6 +37,52 @@ def validate_strain_tasks(self):
 		# Sort the strain tasks based on start days,
 		# maintaining the order for same-day tasks
 		self.cultivation_task.sort(key=lambda task: task.start_day)
+	
+	def update_to_bloomtrace(self):
+		"""
+		Create a new strain on Bloomtrace via Bloomstack.
+		Args: 
+		
+		Returns: 
+			Successful Message to the User
+		"""
+		now = datetime.now()
+		try:
+			data = self.as_dict()
+			data['owner'] = "neil@bloomstack.com"
+			data['modified_by'] = "neil@bloomstack.com"
```

Comment:
this is hard coded, won't work. Has to be the individual who created the file

Reply:
the modified by details should be in "data"

Reply:
it's there but it's coming of the logged in user, which is harsh mule/administrator, and this user is used to get the user_key from admin portal (as per the code) and non of this user can get me the user_key, hence metrc is failing, only neil@bloomstack.com is working.

Reply:
create a user with your email on the admin portal, add the same keys as neil's

Reply:
Cool